tests/models/ctrl/test_modeling_tf_ctrl.py

```diff
@@ -216,33 +216,6 @@ def test_ctrl_sequence_classification_model(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_ctrl_for_sequence_classification(*config_and_inputs)
 
-    def test_model_common_attributes(self):
-        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
-        list_lm_models = [TFCTRLLMHeadModel]
-        list_other_models_with_output_ebd = [TFCTRLForSequenceClassification]
```

Comment:
`TFCTRLForSequenceClassification` incorrectly had a `get_output_embeddings()`, and got the test overridden -- sequence classification models do not have a LM Head (and therefore no output embeddings)